src/source/CppMarshal.scala

```diff
@@ -45,9 +45,13 @@ class CppMarshal(spec: Spec) extends Marshal(spec) {
     case MSet => List(ImportRef("<unordered_set>"))
     case MMap => List(ImportRef("<unordered_map>"))
     case d: MDef => d.defType match {
-      case DEnum | DRecord =>
+      case DEnum | DRecord => List()
```

Comment:
This might be my Scala knowledge being insufficient, but what does it mean to have the empty list here, followed by the longer statement below?  Best I can guess is that the empty list here is redundant, and ends up being evaluated then ignored.